smart-contracts/contracts/dex-router-osmosis/src/contract.rs

```diff
@@ -351,8 +357,13 @@ pub fn simulate_swaps(
     path: Vec<SwapAmountInRoute>,
 ) -> Result<Uint128, ContractError> {
     let querier = PoolmanagerQuerier::new(&deps.querier);
-    let response = querier.estimate_swap_exact_amount_in(0, offer.to_string(), path)?;
-    Ok(Uint128::from_str(&response.token_out_amount)?)
+    let response = querier.estimate_swap_exact_amount_in(0, offer.to_string(), path);
```

Comment:
Please leave a comment about the hardcoded pool_id 0 which could be no-sense for a non conteztualized reader.

Reply:
done